docs/source/plugins.rst

```diff
@@ -243,10 +244,13 @@ by downstream applications.
    `pygeoapi-kubernetes-papermill`_,Euro Data Cube,processes for executing Jupyter notebooks via Kubernetes
    `local-outlier-factor-plugin`_,Manaaki Whenua – Landcare Research,processes for local outlier detection
    `ogc-edc`_,Euro Data Cube,coverage provider atop the EDC API
+   `nldi_xstool`_,United States Geological Survey,Water data processing
```

Comment:
@dblodgett-usgs same here as well, let me know if this is okay or if the wording needs to be updated.

Reply:
That works.